docs/error-backoff.md

```diff
@@ -0,0 +1,94 @@
+# Error Backoff
+
+Error backoff is a mechanism that throttles outgoing TChannel traffic in order to improve the
+health of the Hyperbahn cluster and its hosted services. By definition, error backoff monitors
+the status of outgoing TChannel traffic. If it determines that 1) the Hyperbahn cluster/services
```

Comment:
"the Hyperbahn cluster/services are busy, and 2) the busy state can be improved by throttling outgoing traffic"

Let's only throttle on Busy as first pass.